### DISCUSSION TITLE
Customize Timeline queries rendering

### DISCUSSION THREAD

**Discussion by TristanCacqueray** [srid/emanote#422] [Ideas]
Hello, is it possible to customize a timeline query to include the beginning of the result page, or extra metadata?

For example, how to make:

````
```query {.timeline}
tag:blog/**
```
````

Produce:

```
- YYYY-MM-DD : <post title>
  author: <name> | tags: <list of tags>
  <post intro>

- ...
```

Thanks!

**Comment by srid** [comment]
You can already customize any HTML. In this case, you want to override `query-timeline.tpl` in your notebook. Not sure if all vars are available, but we can add them in a PR if necessary.

**Reply by srid** [reply]
https://emanote.srid.ca/guide/layer
